CalendarFXView/src/main/java/impl/com/calendarfx/view/DayViewEditController.java

```diff
@@ -42,8 +42,10 @@ public DayViewEditController(DayViewBase dayView) {
         dayView.addEventFilter(MouseEvent.MOUSE_PRESSED, this::mousePressed);
         dayView.addEventFilter(MouseEvent.MOUSE_DRAGGED, this::mouseDragged);
         // mouse released is very important for us. register with the scene so we get that in any case.
-        dayView.getScene().addEventFilter(MouseEvent.MOUSE_RELEASED, mouseReleasedHandler);
-        dayView.getScene().addEventFilter(MouseEvent.MOUSE_EXITED, mouseReleasedHandler);
+        if (dayView.getScene() != null) {
+            dayView.getScene().addEventFilter(MouseEvent.MOUSE_RELEASED, mouseReleasedHandler);
```

Comment:
So what if scene is NULL at first. Then the filters will never be set? You should listen to the sceneProperty() of the dayView, too.

Reply:
I do this already just in the next line. I guess, the diff did not show that. This piece is marked as "outdated" in the github gui here.